### PR TITLE
DOC: fix PR01/SA01 errors for pandas.tseries.offsets.*.is_(month|quarter|year)_(end|start)

### PR DESCRIPTION
**Pull request by yuanx749**
- [x] xref #57438 (Replace xxxx with the GitHub issue number)
- [x] xref #57417 (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: doc/source/reference/offset_frequency.rst [hunk 1]
```diff
@@ -26,8 +26,6 @@ Properties
     DateOffset.normalize
     DateOffset.rule_code
     DateOffset.n
-    DateOffset.is_month_start
-    DateOffset.is_month_end
```
Comment (yuanx749) [inline doc/source/reference/offset_frequency.rst:30]: When browsing the preview, I notice these two methods are placed in both Properties and Methods sections.
Seems to be a mistake, so remove them here.

**Review by datapythonista — APPROVED** [review]
Very nice PR @yuanx749, thanks a lot for working on this.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @yuanx749

**Pull request merged**